xbmc/video/dialogs/GUIDialogVideoSettings.cpp

```diff
@@ -169,7 +169,7 @@ void CGUIDialogVideoSettings::Save()
     return;
 
   // prompt user if they are sure
-  if (CGUIDialogYesNo::ShowAndGetInput(12376, 750, 0, 12377))
+  if (CGUIDialogYesNo::ShowAndGetInput(12376, 12377, -1, -1))
```

Comment:
@xhaggi that would be great. Shame we already have so many overload types already. Hope we can trim them back to only a couple.